src/khepri_projection.erl

```diff
@@ -257,9 +263,23 @@ init(#khepri_projection{name = Name, ets_options = EtsOptions}) ->
             {error, exists}
     end.
 
+-spec delete(Projection) -> Ret when
+      Projection :: projection(),
+      Ret :: ok | {error, does_not_exist}.
+%% @hidden
+%% Deletes the ETS table for the given projection.
+%%
+%% @returns `ok' if the table is successfully deleted or `{error,
+%% does_not_exist}' if the table does not exist.
+
 delete(#khepri_projection{name = Name}) ->
-    _ = ets:delete(Name),
-    ok.
+    try
+        ets:delete(Name),
+        ok
+    catch
+        error:badarg:_Stacktrace ->
+            {error, does_not_exist}
+    end.
```

Comment:
Could you please add an `unregister_projections()` test with an ETS table removed behind the scene and check that the Ra server doesn’t crash?